runtime-modules/content/src/tests/migration.rs

```diff
@@ -54,22 +54,104 @@ fn migration_test() {
+            Content::create_channel(
+                Origin::signed(FIRST_MEMBER_ORIGIN),
+                ContentActor::Member(FIRST_MEMBER_ID),
+                ChannelCreationParametersRecord {
+                    assets: None,
+                    meta: Some(vec![]),
+                    reward_account: None,
+                },
+            ),
+            Error::<Test>::MigrationNotFinished
+        );
+        assert_err!(
+            Content::update_channel(
+                Origin::signed(FIRST_MEMBER_ORIGIN),
+                ContentActor::Member(FIRST_MEMBER_ID),
+                <Test as storage::Trait>::ChannelId::one(),
+                ChannelUpdateParametersRecord {
+                    assets_to_upload: None,
+                    new_meta: Some(vec![]),
+                    reward_account: None,
+                    assets_to_remove: BTreeSet::new(),
+                },
+            ),
+            Error::<Test>::MigrationNotFinished
         );
+        assert_err!(
+            Content::update_video(
+                Origin::signed(FIRST_MEMBER_ORIGIN),
+                ContentActor::Member(FIRST_MEMBER_ID),
+                <Test as Trait>::VideoId::one(),
+                VideoUpdateParametersRecord {
+                    assets_to_upload: None,
+                    new_meta: Some(vec![]),
+                    assets_to_remove: BTreeSet::new(),
+                },
+            ),
+            Error::<Test>::MigrationNotFinished
+        );
+        assert_err!(
+            Content::delete_video(
+                Origin::signed(FIRST_MEMBER_ORIGIN),
+                ContentActor::Member(FIRST_MEMBER_ID),
+                <Test as Trait>::VideoId::one(),
+                BTreeSet::new(),
+            ),
+            Error::<Test>::MigrationNotFinished
+        );
+        assert_err!(
+            Content::delete_channel(
+                Origin::signed(FIRST_MEMBER_ORIGIN),
+                ContentActor::Member(FIRST_MEMBER_ID),
+                <Test as storage::Trait>::ChannelId::one(),
+                0u64,
+            ),
+            Error::<Test>::MigrationNotFinished
+        );
+        assert_err!(
+            Content::update_video_censorship_status(
+                Origin::signed(FIRST_MEMBER_ORIGIN),
+                ContentActor::Member(FIRST_MEMBER_ID),
+                <Test as Trait>::VideoId::one(),
+                false,
+                b"test".to_vec()
+            ),
+            Error::<Test>::MigrationNotFinished
+        );
+        assert_err!(
+            Content::update_channel_censorship_status(
+                Origin::signed(FIRST_MEMBER_ORIGIN),
+                ContentActor::Member(FIRST_MEMBER_ID),
+                <Test as storage::Trait>::ChannelId::one(),
+                false,
+                b"test".to_vec()
+            ),
+            Error::<Test>::MigrationNotFinished
+        );
+
+        run_to_block(6); // video migration finished 5 blocks later
```

Comment:
The number of blocks to run to should be a function the number of channels/videos migrated per block as well as the number of videos and channels actually created in the test, to avoid needing to update the value here when those values are changed.